web/source/kmwosk.ts

```diff
@@ -1028,9 +1031,12 @@ if(!window['keyman']['initialized']) {
         keymanweb.domManager.initActiveElement(Lelem);
 
         // Exclude menu and OSK hide keys from normal click processing
-        if(keyName == 'K_LOPT' || keyName == 'K_ROPT')
-        {
-            osk.optionKey(e,keyName,true); return true;
+        if(keyName == 'K_LOPT' || keyName == 'K_ROPT') {
+          osk.optionKey(e,keyName,true);
+          return true;
+        } else if(keyName == 'K_BKSP') {
```

Comment:
... thanks for bringing this up.  I remember seeing a branch for K_BKSP there before and modified it to match the old behavior, but it's not in master.  Not sure how that happened, but it's not a direct correlation to the code that's there now, at the very least.  I'll look into it.

Reply:
For future records:  what I saw was on the touch handler, which originally bypassed this method entirely.  Which _was_ a bug, but adding the `'K_BKSP'` check here was also a bug, since there was code later in the method that could be relied upon.

Reply:
Oh yeah, it's because it was in touch but the recorder needs `clickKey.`  But the implementation in touch was a bug, as we realized after a little discussion.